draft-irtf-cfrg-vdaf.md

```diff
@@ -1582,10 +1582,10 @@ struct {
   MessageType type;
   select (Message.type) {
     case initialize:
-      opaque prep_share<0..2^32-1>;
+      opaque prep_share<1..2^32-1>;
     case continue:
       opaque prep_msg<0..2^32-1>;
-      opaque prep_share<0..2^32-1>;
+      opaque prep_share<1..2^32-1>;
     case finish:
       opaque prep_msg<0..2^32-1>;
   };
```

Comment:
The smallest message is 10 bytes:
* 1 byte for the type discriminator
* 9 bytes for the longest variant, which is continue:
  * 0 bytes for the prep message + 4 bytes for the length prefix
  * 1 byte for the prep share + 4 bytes for the length prefix

Reply:
Following this logic, I think the smallest message is 5 bytes:

* 1 byte for the type discriminator
* 4 bytes for the shortest variant, which is finish:
  * 0 bytes for the prep message + 4 bytes for the length prefix

Reply:
Oofta, you're right. Some times my brain just swaps min/max, $\leq$/$\greq$, $x$/$-x$, etc. Will fix here and in the DAP PR.

Reply:
Noted here that the shortest pingpong message is 5 bytes.